File: CS/WordProcessingFileAPI_CalcDocumentVariable/SampleData.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Collections;

namespace WordProcessingFileAPI_CalcDocumentVariable
{
    class SampleData : ArrayList
    {
        public SampleData()
        {
            Add(new AddresseeRecord("Maria", "Alfreds Futterkiste", "Obere Str. 57, Berlin", "Berlin"));
            Add(new AddresseeRecord("Laurence", "Bon app'", "12, rue des Bouchers, Marseille", "Marseille"));
            Add(new AddresseeRecord("Patricio", "Cactus Comidas para llevar", "Cerrito 333, Buenos Aires", "Buenos Aires"));
            Add(new AddresseeRecord("Thomas", "Around the Horn", "120 Hanover Sq., London", "London"));
            Add(new AddresseeRecord("Boris", "Express Developers", "Krasnoarmeiskiy prospect 25, Tula", "Tula"));
        }
    }

    public class AddresseeRecord
    {
        public string Name { get; set; }
        public string Company { get; set; }
        public string Address { get; set; }
        public string City { get; set; }

        public AddresseeRecord(string _Name, string _Company, string _Address, string _City)
        {
            this.Name = _Name;
            this.Company = _Company;
            this.Address = _Address;
            this.City = _City;
        }
    }
}

File: VB/WordProcessingFileAPI_CalcDocumentVariable/SampleData.vb
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.Collections

Namespace WordProcessingFileAPI_CalcDocumentVariable
	Friend Class SampleData
		Inherits ArrayList

		Public Sub New()
			Add(New AddresseeRecord("Maria", "Alfreds Futterkiste", "Obere Str. 57, Berlin", "Berlin"))
			Add(New AddresseeRecord("Laurence", "Bon app'", "12, rue des Bouchers, Marseille", "Marseille"))
			Add(New AddresseeRecord("Patricio", "Cactus Comidas para llevar", "Cerrito 333, Buenos Aires", "Buenos Aires"))
			Add(New AddresseeRecord("Thomas", "Around the Horn", "120 Hanover Sq., London", "London"))
			Add(New AddresseeRecord("Boris", "Express Developers", "Krasnoarmeiskiy prospect 25, Tula", "Tula"))
		End Sub
	End Class

	Public Class AddresseeRecord
		Public Property Name() As String
		Public Property Company() As String
		Public Property Address() As String
		Public Property City() As String

		Public Sub New(ByVal _Name As String, ByVal _Company As String, ByVal _Address As String, ByVal _City As String)
			Me.Name = _Name
			Me.Company = _Company
			Me.Address = _Address
			Me.City = _City
		End Sub
	End Class
End Namespace
